superset-frontend/src/SqlLab/types.ts

```diff
@@ -49,6 +49,7 @@ export interface QueryEditor {
   selectedText?: string;
   queryLimit?: number;
   description?: string;
+  uuid: string | null;
```

Comment:
Could we use an optional string like the ones above?

Reply:
My understanding is that `uuid: string | null` means the key will always be there, but may be null, and `uuid?: string` means that the key may or may not be there, but if it is it's a string.  This way seemed more accurate to me, I copied from the type def for the `remoteId` here (which is the saved query numerical id), since it should be the same for UUID